CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 
 ### Internals
 * Upgrade pegtl to 2.6.1. Several issues fixed.
+* A bug was fixed in `realm::util::DirScanner` that could cause it to sometimes
```

Comment:
This is not only internal? But affecting a customer?

Reply:
So we should move it and mention the issues: realm/realm-sync#2699.

Reply:
By definition "Internal" is something that doesn't affect any users of the component - i.e. a refactoring, a test etc.

Reply:
I thought it meant for other Realm developers, which is the policy that has previously been communicated. I'm fine with moving it if you think.

Reply:
Core doesn't have multiple layers of "Internal" as in Sync :-)
Here it's just purely internal - to Core :-)
So moving it would be great.